### ISSUE TITLE
Question: do you plan to integrate it into telegraf?

### ISSUE THREAD

**Issue by gaetanquentin** [Oxalide/vsphere-influxdb-go#61]
Do you plan to integrate this good tool as a plugin into telegraf (written oin go too) ?

**Comment by sofixa** [comment]
Hi, 

That was my original idea but i had a hard time going around some issues(mostly how to correlate the telegraf interval and vSphere's metric collection interval, plus different development cycles). Currently at work we use it with telegraf as a relay(vsphere-influxdb-go executes every minute and sends to a telegraf tcp listener in InfluxDB format, which then relays based on tags to a few InfluxDBs) to profit some of telegraf's nice features(caching, easy filtering/sharding).

Now that the tool is more or less stable(there are some feature requests and bugs, but general basic functionality is there), it might be a good idea to split the vSphere collection functions into a library, and add it as a telegraf plugin while also maintaining this project as a standalone tool without code duplication. However, i don't have a lot of free time currently, and i prefer to spend it in adding extra features and fixing #55 which is proving to be a hard one to tackle. 

TL;DR: I think i might be a good idea, but it will require some refactoring and i don't have the time now, but might look into it in the future

Cheers,
Adrian
